packages/e2e-tests/specs/experiments/widgets.test.js

```diff
@@ -0,0 +1,363 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	visitAdminPage,
+	deactivatePlugin,
+	activatePlugin,
+	activateTheme,
+} from '@wordpress/e2e-test-utils';
+
+/** @typedef {import('puppeteer').ElementHandle} ElementHandle */
+
+describe( 'Widgets screen', () => {
+	beforeEach( async () => {
+		await visitAdminPage( 'themes.php', 'page=gutenberg-widgets' );
+		// Wait for the widget areas to load.
+		await page.waitForSelector(
+			'[aria-label="Block: Widget Area"][role="group"]'
+		);
+	} );
+
+	afterEach( async () => {
+		await cleanupWidgets();
+	} );
+
+	beforeAll( async () => {
+		// TODO: Ideally we can bundle our test theme directly in the repo.
```

Comment:
There's some prior art here with the `tt1-blocks` theme which is used to test FSE.

https://github.com/WordPress/gutenberg/blob/d61dccc58235a33f1caa00f2c396979d1691abb0/.wp-env.json#L6

We can look into it later, though. It's more important that we get _some_ tests for widgets in right now 🙂

Reply:
Yeah, I'm aware of that, but figured we should do it in a following PR 👍 .